<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ticket_add_form</class>
 <widget class="QWidget" name="Ticket_add_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление билета</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>261</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="tour_ID">
      <property name="text">
       <string>ID тура</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="tourist_ID">
      <property name="text">
       <string>ID туриста</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="date_purchase">
      <property name="text">
       <string>Дата покупки</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="tour_ID_input"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="tourist_ID_input"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="date_purchase_input"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="addition">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="ticket_ID">
      <property name="text">
       <string>ID билета</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="ticket_ID_input"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
